### ISSUE TITLE
Unable to find target command: compile_sass

### ISSUE THREAD

**Issue by knkevinlai** [blitzrk/sublime_libsass#6]
Many thanks for your work on it. I am impressed by the built-in compiler.

But I am not able to get libsass to work. The console show 'Unable to find target command: compile_sass' when I press "CMD" + "B" to build.

Related info: 
Package "SASS" has been installed as well.
Sublime text 2.0.2 on Mac OS X 10.10.2

Thank you.
Kevin

**Comment by blitzrk** [comment]
Thanks for letting me know! I had named a directory `...darwin...` instead of `...osx...`. Cross-compiling must have gotten me all confuzzled.

Anyway, I fixed it and pushed a new release. It may take an hour for package control to see the new version, but if you close and reopen Sublime in about an hour, it should automatically upgrade and work for you.

Let me know if you have any other problems or any suggestions (in a new issue) once you get it working!
